plugins/civo/api_key.go

```diff
@@ -0,0 +1,84 @@
+package civo
```

Comment:
Please take this out, looks like it made its way in this PR again 😄

Reply:
This file is no longer on the PR. Marking as resolved.